CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
+# v0.25.0 Release - 3/15/2019
+
+*Note*: This release comes with a config change, use `skaffold fix` to permanently upgrade your config to `v1beta7`, however old versions are now auto-upgraded. 
+See [deprecation-policy.md](/deprecation-policy.md) for details on what beta means.
+
+
+*Deprecation notice*: With this release we mark for deprecation the `additionalFlags` field in kaniko, instead the kaniko builder plugin .
```

Comment:
whoops, looks like you may have cut off the sentence by accident, maybe: 

With this release we mark for deprecation the `additionalFlags` field in kaniko; instead any additional flags will now be represented as unique fields in the KanikoArtifact type.